### ISSUE TITLE
Remove Aegis from Helm charts.

### ISSUE THREAD

**Issue by v0lkan** [vmware-tanzu/secrets-manager#1]
Helm charts file have the Aegis header.

They have to be replaced with the following as per VMware OSPO guidance.

```text
# /*
# |    Protect your secrets, protect your sensitive data.
# :    Explore VMware Secrets Manager docs at https://vsecm.com/
# </
# <>/  keep your secrets… secret
# >/
# <>/' Copyright 2023–present VMware, Inc.
# >/'  SPDX-License-Identifier: BSD-2-Clause
# */
```

In addition, the current repository user is `vsecm`
so any `aegishub` repo will now be `vsecm`

Here’s an example

`aegishub/aegis-ist-safe` will become => `vsecm/vsecm-ist-safe`

**Comment by abhishek44sharma** [comment]
Fixed by #7

**Comment by abhishek44sharma** [comment]
Fixed by #64
